### PR TITLE
Feature/new share api

### PR DESCRIPTION
**Pull request by beshkenadze**
**What's works:** 

1. Share the personal items to the members

**What's doesn't work**

1.  ~~Do not update list of the user for the shared item until the app will be reloaded~~
2.  ~~After create the item the permissions of that item doesn't work until the app will be reloaded~~
3. Do not create the new members accounts if shared user doesn't exsist.
4. Invited users do not showing in the share modal.

### CODE REVIEW

File: packages/components/MenuList/components/MenuListInner.js [hunk 1]
```diff
@@ -133,9 +133,9 @@ const MenuListInnerComponent = ({
   const menuList = [
     {
       id: isPersonal ? personalLists.inbox?.id : null,
-      title: 'Inbox',
+      title: 'Shared with me',
```
Comment (cu4xxi) [inline packages/components/MenuList/components/MenuListInner.js:136]: In the middle column is still Inbox title.
Maybe fix it in selector 'inboxListSelector'?

File: packages/containers/Team/Team.js [hunk 2]
```diff
@@ -134,6 +134,7 @@ export const TeamContainer = () => {
   };
 
   const handleInvite = invitedMembers => {
+    debugger;
```
Comment (cu4xxi) [inline packages/containers/Team/Team.js:137]: pls, remove debugger :)

File: packages/components/MenuList/components/MenuListInner.js (hunk 1 above)
Comment (beshkenadze) [reply packages/components/MenuList/components/MenuListInner.js:136]: @cu4xxi in the code the menuList has priotity over the server data, idkw. Btw, works for me.

File: packages/containers/Team/Team.js (hunk 2 above)
Comment (beshkenadze) [reply packages/containers/Team/Team.js:137]: My bad :(

File: packages/components/MenuList/components/MenuListInner.js (hunk 1 above)
Comment (cu4xxi) [reply packages/components/MenuList/components/MenuListInner.js:136]: @beshkenadze 
you're talking about menuList.
But I am about MiddleColumn.
I've reproduce such case with favorites (see screenshot)
https://scr.4xxi.com/cu/Dashboard_2020-10-15_3_PM-29-19.png

**Review by cu4xxi — APPROVED** [review]

**Pull request merged**